packages/app/src/cli/models/extensions/load-specifications.ts

```diff
@@ -24,15 +22,6 @@ const SORTED_CONFIGURATION_SPEC_IDENTIFIERS = [
   AppHomeSpecIdentifier,
 ]
 
-/**
- * Load all specifications from the local file system AND plugins
- */
-export async function loadLocalExtensionsSpecifications(config: Config): Promise<ExtensionSpecification[]> {
```

Comment:
why not keeping this method and removing the call to `loadUIExtensionSpecificationsFromPlugins`? It make sense to me calling the `loadLocalExtensionsSpecifications` method. You avoid lot of changes in the PR :)

Reply:
that makes senes 👌